packages/components/modules/notifications/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const NOTIFICATION_VERB = {
+  commentCreated: 'COMMENTS.COMMENT_CREATED',
```

Comment:
why did we change that?

Reply:
This change is based on the changes of this PR https://github.com/silverlogic/baseapp-backend/pull/132